lib/commands/doctor/checks/system-stack.js

```diff
@@ -35,24 +37,35 @@ function systemStack(ctx) {
         });
     }
 
-    return promise.then(() => { return {yes: true}; }).catch((error) => {
+    return promise.catch((error) => {
         ctx.ui.log(
             `System checks failed with message: '${error.message}'
 Some features of Ghost-CLI may not work without additional configuration.
 For local installs we recommend using \`ghost install local\` instead.`,
             'yellow'
         );
 
-        return ctx.ui.allowPrompt ? ctx.ui.confirm(chalk.blue('Continue anyway?'), false) : Promise.resolve({yes: false});
-    }).then(answer => answer.yes || Promise.reject(
-        new errors.SystemError('System checks failed.')
-    ));
+        return (ctx.ui.allowPrompt ?
+            ctx.ui.confirm(chalk.blue('Continue anyway?'), false) :
+            Promise.resolve({yes: false})
+        ).then((answer) => {
+            if (answer.yes) {
+                task.skip('System stack check skipped');
+                return Promise.resolve();
+            }
+
+            return Promise.reject(new errors.SystemError({
+                message: `System stack checks failed with message: '${error.message}'`,
+                task: 'Checking operating system compatibility'
+            }));
+        });
+    });
 }
 
 module.exports = {
-    title: 'Checking operating system compatibility',
+    title: taskTitle,
     task: systemStack,
     enabled: (ctx) => !ctx.local,
-    skip: (ctx) => ctx.argv && !ctx.argv.stack,
+    skip: (ctx) => !ctx.isDoctorCommand && ctx.argv && !ctx.argv.stack,
```

Comment:
What is the diff between `ctx.local` and `argv.stack`? 🤔 
Aren't they the same somehow? `argv.stack` is false if local PM is used.

Reply:
Discussed.

`--no-stack` => `argv.stack=false` (but can be local PM or a different extension)